src/bundle/Resources/views/themes/admin/ui/component/middle_ellipsis/middle_ellipsis.html.twig

```diff
@@ -1,4 +1,6 @@
-<span class="ibexa-middle-ellipsis" title="{{ name }}">
+{% set is_deletable = is_deletable is defined ? is_deletable : true %}
+
+<span class="ibexa-middle-ellipsis" title="{{ name }} {% if not is_deletable %} not-deletable{% endif %}">
```

Comment:
I don't think it is a good idea to add something to a "title", title can be HTML so in some cases this might be a problem